flow-typed/forms.js

```diff
@@ -283,7 +284,16 @@ declare type PageElement = _FormElementBase & {
   label: string,
 } & _NestedElementsElement
 
-declare type NestedElementsElement = PageElement | RepeatableSetElement
+declare type SectionElement = _FormElementBase & {
```

Comment:
Can `_NestedElementsElement` inherit `_FormElementBase`?

Reply:
No, some use `FormElementBase` , some use `_FormElementBase`. Better this way IMO